saleor/webhook/validators.py

```diff
@@ -24,6 +23,7 @@ def custom_headers_validator(headers: Dict[str, str]) -> Dict[str, str]:
             f"Number of headers exceeds the limit: {HEADERS_NUMBER_LIMIT}."
         )
 
+    headers = {k.lower(): v for k, v in headers.items()}
```

Comment:
In line 28, you're checking if the `key` is `str`, but before this you're calling `.lower()` which means, that this can fail, right?